README.md

```diff
@@ -1,53 +1,88 @@
+📚 Pretty interface for Markdown docs \
+⤴️ Lets you open PRs for wiki docs
 
-## Before using this action
+## Usage
+
+❗ Make sure you turn on the GitHub wiki feature in your repo's settings menu.
+You'll also need to _manually_ create a dummy page to initialize the git repo
+that powers the GitHub wiki. If you don't, when we push to `your-repo.wiki.git`,
+your workflow will fail because the wiki doesn't exist.
+
+<div align="center">
 
-You probably want to commit your wiki in its initial state before using this action.
-This is because this action will commit *all* differences between the wiki and the
-`.github/wiki` (or other specified directory) contents.
+![](https://user-images.githubusercontent.com/61068799/225441831-d3587ceb-0462-4591-bf00-ee56b040fe00.png)
 
-```bash
-git clone https://github.com/octocat/my-repo.wiki.git .github/wiki
-rm -rf .github/wiki/.git
-git add .github/wiki
-git commit -m "Add wiki to repo"
+</div>
+
+Create a workflow file named something like `.github/workflows/wiki.yml` and add
+a job that uses this action to it! 🎉
+
+```yml
+name: Publish wiki
+on:
+  push:
+    branches: [main]
+    paths: [wiki/**, .github/workflows/publish-wiki.yml]
+concurrency:
+  group: wiki
+  cancel-in-progress: true
+permissions:
+  contents: write
+jobs:
+  wiki:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: spenserblack/actions-wiki@v0.1.1
 ```
 
-## Usage
+⚠️ Make sure you enable the `content: write` permission! GitHub recently
+[changed the default permissions granted to jobs] for new repositories.
 
-Unfortunately, `${{ github.token }}` does not have access to updating wiki
-repositories, so it is necessary to create a new PAT.
+☁️ If you're pushing to a wiki that's **not the current repository** you'll need
+to get a [GitHub PAT] to push to it.
 
-Start by creating a classic token. If your repository is public, you only need the
-`public_repo` scope. Otherwise, you must grant the full `repo` scope. Then, add
-the token to your repository secrets so that it can be used in your workflow.
+### Options
 
-Simple example usage can be found in [this workflow file](./.github/workflows/update-wiki.yml).
+- **`repository`:** The repository housing the wiki. Use this if you're
+  publishing to a wiki that's not the current repository. You _can include a
+  domain prefix_ if you have a hosted GitHub instance that you want to push to.
+  Default is `${{ github.repository }}` with the implicit `github.com` domain.
+
+- **`token`:** `${{ github.token }}` is the default. This token is used when
+  cloning and pushing wiki changes.
+
+- **`wiki-directory`:** The directory to use for your wiki contents. Default:
+  `.github/wiki`.
+
+- **`commit-message`:** The message to use when committing new content. Default
+  is `Update wiki ${{ github.sha }}`. You probably don't need to change this,
+  since this only applies if you look _really closely_ in your wiki.
```

Comment:
Per our discussions, what do you think about moving these to be comments in the usage code block?

Reply:
Yep! It's either this list or the code block. I'll bow to whatever gets decided in #20 .

In this case, I just wanted to see how the entire thing looked with this `<ul>` thing. I think it honestly looks pretty good, fits semantically, works for screen readers better than "this is the token that we use _pause_ **pound sign** to do things..." does the job of elaborating on options, and maintains the markdown purity that you seem to crave 😋.

Reply:
To be resolved by @jcbhmr using _both_ an actions `<ul>` and comments in the copy-paste example